### PR TITLE
Use new way to access category images

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Changes template to use a new way of accessing category images, prepared in https://github.com/PrestaShop/PrestaShop/pull/32653. After this is merged, we can merge the final part https://github.com/PrestaShop/PrestaShop/pull/36877, tests should work.
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     |
| Sponsor company   | 
| How to test?      | There should be no visible effect in functionality.

### How to test
- Basically, everything should work the same.
- Check category cover image:
  - Visit a category with a cover image, there should be an image next to category description.
  - Visit a category with NO cover image, there should be NO image next to category description. There should be NO placeholder `Image not found image`.
- Check category thumbnail:
  - Assign thumbnail to `Men` category.
  - Assign NO images to `Women` category.
  - Visit `Clothes`. In the subcategories, `Men` should have an image, `Women` should have placeholder `Image not found image`.

### CODE REVIEW

**Review by ShaiMagal — APPROVED** [review]

**Comment by ShaiMagal** [comment]
@florine2623 ZIP build fail is not related. :-)

**Review by florine2623 — APPROVED** [review]
Hello @Hlavtox ,

Works as expected.

Screenshot 2024-09-17 at 15 03 08
Screenshot 2024-09-17 at 15 03 32


LGTM, it is QA ✅

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
